build.cmd

```diff
@@ -2,24 +2,28 @@
 setlocal
```

Comment:
The changes to build.cmd are unrelated to this PR.

Reply:
Removed.